conans/test/conftest.py

```diff
@@ -47,7 +48,9 @@
         "0.28": {
             "path": {
                 # Using chocolatey in Windows -> choco install pkgconfiglite --version 0.28
-                'Windows': "C:/ProgramData/chocolatey/lib/pkgconfiglite/tools/pkg-config-lite-0.28-1/bin"
+                'Windows': "C:/ProgramData/chocolatey/lib/pkgconfiglite/tools/pkg-config-lite-0.28-1/bin",
+                'Darwin': "/opt/homebrew/bin/pkg-config" if MacOS_M1 else "/usr/local/bin/pkg-config",
+                'Linux': "/usr/bin/pkg-config"
```

Comment:
It was a bug in our test suite. All the tests specifying that tool were skipping MacOS and Linux.